Tests/CommandsTests/PackageCommandTests.swift

```diff
@@ -49,8 +49,9 @@ final class PackageCommandTests: CommandsTestCase {
     }
 
     func testUsage() async throws {
+        throw XCTSkip("rdar://131126477")
```

Comment:
Why? The reason for skipping is opaque to non apple contributors without a comment

Reply:
There is a regression in ArgumentParser 1.4.0 that won't allow this test to pass.

Reply:
Sure but what's the failure, because if it's noted in source and linked to a GitHub issue then others can contribute more easily.